packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -38,7 +67,13 @@ class Config(_BaseConfig):
                     "type": "string",
                     "value": ["node.platform.os == linux"],
                 },
-                # resources
+                # SEE service_settings_labels.py::ContainerSpec
+                {
+                    "name": "ContainerSpec",
+                    "type": "ContainerSpec",
```

Comment:
not sure I remember very well what I tried to do, but should the type not be "Command"? This is what defined the array right?

Reply:
no, the type is ``ContainerSpec`` and has a field called ``Command``
```yaml
io.simcore.type: '{"type": "computational"}'     
simcore.service.settings: '[{"name": "ContainerSpec", "type": "ContainerSpec", "value": {"Command": ["prog", "arg1"]}}]'
#...
```

Reply:
let's discuss that one

Reply:
discussed